development-docs/RELEASE.md

```diff
@@ -39,7 +39,7 @@ The release process should normally look like this:
   * Copy the `helm-charts/index.yaml` from the `release` branch to `master`.
 
 12. _(only for GA, not for RCs)_ Update the Strimzi manifest files in Operate Hub [community operators](https://github.com/operator-framework/community-operators) repository and submit a pull request upstream. *Note*: Instructions for this step need updating.
-13. _(only for GA, not for RCs)_ Add the new version to the `systemtest/src/main/resources/StrimziUpgradeST.json` file for the upgrade tests
+13. _(only for GA, not for RCs)_ Add the new version to the `systemtest/src/test/resources/upgrade/StrimziUpgradeST.json` file for the upgrade tests
```

Comment:
I guess we should add the StrimziDowngradeST.json here as well?

Reply:
I will add it